README.rst

```diff
@@ -109,6 +109,15 @@ In order to run the server execute the following command::
 
     make serve
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Piping data to Elasticsearch
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Before starting the server, set the following environment variables::
+
```

Comment:
Do you want to mention anything here about the docker compose file. Steps to get it up and running?

Reply:
Done